File: demo/src/pages/LabelDemo/LabelDemo.ts
import { main, Viewable, Circle, Text, Label } from "@tswift/ui";

@main
export class LabelDemo extends Viewable {

    body = Label({
        icon: Circle()
            .fill('.blue')
            .frame({ width: 44, height: 44, alignment: '.center' })
            .overlay(Text("BL"))
    },

        Text("Bob Loblaw")
            .font('.body')
            .foregroundColor('.primary'),
        Text("Person at Law")
            .font('.subheadline')
            .foregroundColor('.secondary')
    )
}

File: demo/src/pages/LabelDemo/LabelDemo.swift
import SwiftUI

struct LabelDemo: View {
    var body: some View {
        Label {
            Text("Bob Loblaw")
                .font(.body)
                .foregroundColor(.primary)
            Text("Person at Law")
                .font(.subheadline)
                .foregroundColor(.secondary)
        } icon: {
            Circle()
                .fill(.blue)
                .frame(width: 44, height: 44, alignment: .center)
                .overlay(Text("BL"))
        }
    }
}

struct LabelDemo_Previews: PreviewProvider {
    static var previews: some View {
        LabelDemo()
    }
}
